config/initializers/dqt.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "dqt"
+
+Dqt.configure do |config|
+  def json_to_hash(json)
+    JSON.parse(json.gsub("=>", ":"), symbolize_names: true)
+  end
+
+  config.client.headers = { Authorization: Rails.application.config.dqt_client_api_key } if Rails.application.config.respond_to?(:dqt_client_api_key)
+  config.client.host = Rails.application.config.dqt_client_host if Rails.application.config.respond_to?(:dqt_client_host)
+  config.client.params = json_to_hash(Rails.application.config.dqt_client_params) if Rails.application.config.respond_to?(:dqt_client_params) && Rails.application.config.dqt_client_params.present?
```

Comment:
Should we check that we have the right config first, and set everything later? Does having partial config give us anything?

Also, just to make sure, this `config.client` only affects the dqt client, right? Won't do anything to our other clients?

Reply:
> Also, just to make sure, this config.client only affects the dqt client, right? Won't do anything to our other clients?
correct